google/cloud/datastore_admin_v1/gapic/datastore_admin_client.py

```diff
@@ -17,7 +17,7 @@
 """Accesses the google.datastore.admin.v1 DatastoreAdmin API."""
 
 import functools
-import pkg_resources
+import os
```

Comment:
Did we resolve the issue of changing macro-generated code here?  We are going to be bombarded by autosynth.

Reply:
E.g., PR #105 wants to undo this change.